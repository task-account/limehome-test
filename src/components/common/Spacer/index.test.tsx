import renderer from "react-test-renderer";
import { Spacer } from ".";

describe(`Spacer`, () => {
  it("render", () => {
    const tree = renderer.create(<Spacer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("vertical", () => {
    const tree = renderer.create(<Spacer vertical={"l"} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("horizontal", () => {
    const tree = renderer.create(<Spacer horizontal={"l"} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
