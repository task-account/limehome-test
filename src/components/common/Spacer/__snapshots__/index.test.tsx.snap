// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer horizontal 1`] = `
<View
  style={
    Object {
      "flex": undefined,
      "marginBottom": undefined,
      "marginHorizontal": 24,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "marginVertical": undefined,
    }
  }
/>
`;

exports[`Spacer render 1`] = `
<View
  style={
    Object {
      "flex": undefined,
      "marginBottom": undefined,
      "marginHorizontal": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "marginVertical": undefined,
    }
  }
/>
`;

exports[`Spacer vertical 1`] = `
<View
  style={
    Object {
      "flex": undefined,
      "marginBottom": undefined,
      "marginHorizontal": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "marginVertical": 24,
    }
  }
/>
`;
